yente/routers/search.py

```diff
@@ -217,19 +217,19 @@ async def fetch_entity(
 
 @router.get(
     "/entities/{entity_id}/adjacent",
+    name="Fetch Adjacent Entities*",
     tags=["Data access"],
     response_model=EntityAdjacentResponse,
     responses={
-        307: {"description": "The entity was merged into another ID"},
+        308: {"description": "The entity was merged into another ID"},
         404: {"model": ErrorResponse, "description": "Entity not found"},
         500: {"model": ErrorResponse, "description": "Server error"},
     },
-    include_in_schema=False,  # Hide while we try it out.
 )
 async def fetch_adjacent_entities(
     response: Response,
     entity_id: str = Path(
-        description="ID of the entity to retrieve", examples=["Q7747"]
+        description="ID of the entity whose peers are requested", examples=["Q7747"]
```

Comment:
whose graph context was requested? peers introduces a new technical term we haven't used yet.